eth_tester/backends/pyevm/main.py

```diff
@@ -507,6 +511,16 @@ def call(self, transaction, block_number="latest"):
             block_number,
         )
         if computation.is_error:
-            raise TransactionFailed(str(computation._error))
+            msg = str(computation._error)
+
+            # Check to see if it's a EIP838 standard error, with ABI signature
+            # of Error(string). If so - extract the message/reason.
+            if isinstance(computation._error, EVMRevert) and \
```

Comment:
We've largely avoided this style of line continuation.  My suggestion would be to move this to a small utility function.

```python
def is_eip838_error(error):
    if not isinstance(err, EVMRevert):
        return False
    elif ...:
        ...
```

This will allow you to add unit tests for this specific logic which I'd like to see added.

Then in this function body you can do

```python
if is_eip838_error(computation._error):
    ...
```